src/celeritas/neutron/model/ChipsNeutronElasticModel.cc

```diff
@@ -0,0 +1,299 @@
+        coeffs.par[28] = 64 / a3;
+        // The gloria pre-exponent (pel_us)
+        coeffs.par[29] = 1e8 * std::exp(0.32 * asa);
+        coeffs.par[30] = 20 * std::exp(0.45 * asa);
+        coeffs.par[31] = 7e3 + 2.4e6 / a5;
+        coeffs.par[32] = 2.5e5 * std::exp(0.085 * a3);
+        coeffs.par[33] = 2.5 * a;
+        // The gloria slope (pel_ub)
+        coeffs.par[34] = 920 + 0.03 * a8 * a3;
+        coeffs.par[35] = 93 + 2.3e-3 * a12;
+    }
+    else
+    {
+        // The main pre-exponent (peh_sg)
+        coeffs.par[0] = 4.5 * std::pow(a, 1.15);
+        coeffs.par[1] = 0.06 * std::pow(a, 0.6);
+        coeffs.par[2] = 0.6 * a / (1 + 2e15 / a16);
+        coeffs.par[3] = 0.17 / (a + 9e5 / a3 + 1.5e33 / a32);
+        coeffs.par[4] = (1e-3 + 7e-11 * a5) / (1 + 4.4e-11 * a5);
+        coeffs.par[5] = (ipow<2>(2.2e-28 * a10) + 2e-29) / (1 + 2e-22 * a12);
+        // The main slope (peh_sl)
+        coeffs.par[6] = 400 / a12 + 2e-22 * a9;
+        coeffs.par[7] = 1e-32 * a12 / (1 + 5e22 / a14);
+        coeffs.par[8] = 1e3 / a2 + 9.5 * sa * std::sqrt(sa);
+        coeffs.par[9] = 4e-6 * a * asa + 1e11 / a16;
+        coeffs.par[10] = (120 / a + 2e-3 * a2) / (1 + 2e14 / a16);
+        coeffs.par[11] = 9 + 100 / a;
+        // The main quadratic (peh_sh)
+        coeffs.par[12] = 2e-3 * a3 + 3e7 / a6;
+        coeffs.par[13] = 7e-15 * a4 * asa;
+        coeffs.par[14] = 9e3 / a4;
+        // The 1st max pre-exponent (peh_qq)
+        coeffs.par[15] = 1.1e-3 * asa / (1 + 3e34 / a32 / a4);
+        coeffs.par[16] = 1e-5 * a2 + 2e14 / a16;
+        coeffs.par[17] = 1.2e-11 * a2 / (1 + 1.5e19 / a12);
+        coeffs.par[18] = 0.016 * asa / (1 + 5e16 / a16);
+        // The 1st max slope (peh_qs)
+        coeffs.par[19] = 2e-3 * a4 / (1 + 7e7 / std::pow(a - 6.83, 14));
+        coeffs.par[20] = 2e6 / a6 + 7.2 / std::pow(a, 0.11);
+        coeffs.par[21] = 11 * a3 / (1 + 7e23 / a16 / a8);
+        coeffs.par[22] = 100 / asa;
+        // The 2nd max pre-exponent (peh_ss)
+        coeffs.par[23] = (0.1 + 4.4e-5 * a2) / (1 + 5e5 / a4);
+        coeffs.par[24] = 3.5e-4 * a2 / (1 + 1e8 / a8);
+        coeffs.par[25] = 1.3 + 3e5 / a4;
+        coeffs.par[26] = 500 / (a2 + 50) + 3;
+        coeffs.par[27] = 1e-9 / a + ipow<4>(6e14 / a16);
+        // The 2nd max slope (peh_sb)
+        coeffs.par[28] = 0.4 * asa + 3e-9 * a6;
+        coeffs.par[29] = 5e-4 * a5;
+        coeffs.par[30] = 2e-3 * a5;
+        coeffs.par[31] = 10;  // p4
+        // The effective pre-exponent (peh_us)
+        coeffs.par[32] = 0.05 + 5e-3 * a;
+        coeffs.par[33] = 7e-8 / sa;
+        coeffs.par[34] = 0.8 * sa;
+        coeffs.par[35] = 0.02 * sa;
+        coeffs.par[36] = 1e8 / a3;
+        coeffs.par[37] = 3e32 / (a32 + 1e32);
+        // The effective slope (peh_ub)
+        coeffs.par[38] = 24;
+        coeffs.par[39] = 20 / sa;
+        coeffs.par[40] = 7e3 * a / (sa + 1);
+        coeffs.par[41] = 900 * sa / (1 + 500 / a3);
+    }
```

Comment:
It looks like the heavy and light nuclei have entirely different parameterizations? Maybe then it would be good to store them in two separate data types so that you can add semantics:
```c++
struct ChipsDiffXsLightCoefficients
{
     Array<real_type, 6> main_preex;
     Array<real_type, 6> main_slope;
     Array<real_type, 6> main_quadratic;
....
};
```

Reply:
Okay.  I will think about it, but again let's keep them as they are for now - may be more convenient for direct comparisons to Geant4 at this stage (which I actually verified all values with a couple a's between this code and Geant4) - just as a sanity check.